west.yml

```diff
@@ -98,7 +98,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 727b36cfe9e2f2aa5eb88d608e6334c8d548cee9
+      revision: zephyrproject-rtos/hal_nxp/pull/53
```

Comment:
The hal pr has been merged, so you can now update this to the new SHA

Reply:
Updated to fix merge conflicts and to include the right commit revision in west.yml